kotlin-native/Interop/Runtime/src/main/kotlin/kotlinx/cinterop/Utils.kt

```diff
@@ -727,3 +735,43 @@ public fun COpaquePointer.readBytes(count: Int): ByteArray {
     nativeMemUtils.getByteArray(this.reinterpret<ByteVar>().pointed, result, count)
     return result
 }
+
+@ExperimentalForeignApi
+public fun setMemory(mem: NativePointed, value: Byte, size: Long) {
+    nativeMemUtils.memset(mem, value, size)
+}
+
+@ExperimentalForeignApi
+public fun copyMemory(destMem: NativePointed, srcMem: NativePointed, size: Long) {
```

Comment:
Do we really need both moveMemory and copyMemory functions? The second one is less safe and performance benefits might not worth it.

Reply:
No real reason i added memmove apart from "might as well complete the set of three" when i was reading the LLVM manual to look up the copy intrinsic. Can remove if you want :)